R/distribution_formulas.R

```diff
@@ -0,0 +1,192 @@
+#' Set up a new distribution for a data type
+#'
+#' @param data_type A string specifying the type of data that the distribution
+#'   will be fit to. Options are listed in the function call, where the first
+#'   option listed, i.e., `"index"` is the default.
+#' @param module An identifier to a C++ fleet module that is linked to the data
+#'   of interest.
+#' @param family A description of the error distribution and link function to
+#'   be used in the model.
+#' @param sd A list of length two. The first entry, `"value"`, stores the
+#'   initial values for the relevant standard deviations. The second entry,
+#'   `"estimated"` is a vector of booleans indicating whether or not
+#'   standard deviation is estimated.
```

Comment:
It is not clear if estimated should be a vector if sd is a vector with a length greater than one.

Reply:
Maybe check if the user provide both `value` and `estimated`, and specify default values in the Roxygen documentation?

Reply:
added to documentation